status-im/status-react
# Adds ATT tokens for easy testing on Ropsten

Fixes #3869

### Summary:

This PR changes the address for ATT, which I redeployed with some additional logic that will send 1000 `ATT` to anyone who sends a 0 ETH txn to the contract address, which is:

0x00a8e52df8f4f1f4b67bded9ae6090b35489a973

I also added Aragon logos to make it look purrrty.

### Steps to test:
- Open Status
- Ensure you are on Ropsten
- Navigate to the wallet, see if you can see the ATT address and send a 0 ETH txn to it.
- Make sure your balance of ATT increases by 1000.

status: ready
